flask/app/templates/play.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+{% set active_page = "index" %}
+{% block title %} Game {% endblock %}
+{% block content %}
+<div class="container">
+    <h1>Rock! Paper! Scissors! Showdown </h1>
+    <div class="flex-container">
+        <img src="./static/img/Woman.png" alt="Avatar1"  id="choice-player" style="border-radius:50%; height: 300px; width: 300px;">
+        <p style="margin-top: 150px; font-size:40px" id="status">VS</p>
+        <img src="./static/img/Man.png" alt="Avatar2" id="choice-opponent" style="border-radius:50%; height: 300px; width: 300px;">
+    </div>
+    <div class="flex-container">    
+        <span class="circle-dot">
+            <img src="./static/img/icons8-rock-80.png" id="btn-rock"></img>
+        </span>
+        <span class="circle-dot">
+            <img src="./static/img/icons8-paper-80.png" id="btn-paper"></img>
+        </span>
+        <span class="circle-dot">
+            <img src="./static/img/icons8-hand-scissors-80.png" id="btn-scissors"></img>
+        </span>
+    </div>
+    <button type="button" class="play-anon-btn" id="btn-play-again" style="border-radius: 15px;">Play Again?</button>
+</div>
+{% endblock %}
```

Comment:
Nice work Hamdia, is it possible for you to add a ref to the script file I'm about to include

I'll put it at /app/static/js/game.js

thanks